### PR TITLE
DOC: add color cycle map into readme

### LINKED ISSUE #11: plot a gallery of the colors in bg_style readme
@dragonyanglong would it also be possible to plot the ``bg_style`` colors in the readme?  I think code to do it could be stolen from here:
https://matplotlib.org/3.1.0/gallery/color/named_colors.html
so it could be quick.   It would certainly be very useful.  For the label for each color, maybe just give its number in the color list?

### PR DESCRIPTION
**Pull request by dragonyanglong**
@sbillinge , this closes #11

### CODE REVIEW

**Comment by sbillinge** [comment]
Thanks @dragonyanglong .  Is it possible to add the code that builds this figure to the repo so that we can reproduce it in the future if we add different color cycles, for example?  Thanks.

**Comment by dragonyanglong** [comment]
Sure thing! I pushed. Please review and merge.

File: example/color_cycle.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import matplotlib.pyplot as plt
+from diffpy.utils.parsers.loaddata import loadData
+from billinge_style.bg_plt_style import bg_plt_style
+import numpy as np
+#if use stylesheet remotely
```
Comment (sbillinge) [inline example/color_cycle.py:5]: clean this.  Also, maybe add instructions how to use it, which will basically be to read the readme and install the package etc.

File: example/color_cycle.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+for (i,c) in enumerate(cycle):
+	ax.plot(x,y+offset*i, label=str(c), color=c,linestyle='-')
+
+ax.set_xlim(0, 2.0) # set x-axis lower and upper limits
+
+ax.legend(loc = 'upper right', frameon = False, prop={'size':14},ncol=1)
+
+
+plt.show()
+# fig.savefig('color_cycle.png')
```
Comment (sbillinge) [inline example/color_cycle.py:31]: need a trailing line-feed?

**Review by sbillinge — COMMENTED** [review]
nice.  Nearly there.   Also, what is the plot.png, is it needed?  or just color_cycle.png

**Comment by dragonyanglong** [comment]
Both are needed. `plot.png` is for the standard plot for our PDF plot (exp, calc, and diff three curves, and axis labels etc) and `color_cycle.png` is for showing the full cycle map directly. Both are shown in the readme.

I updated per your comments and pushed.

**Comment by sbillinge** [comment]
@dragonyanglong ok, got it .  Thanks so much.

**Pull request merged**